CHANGELOG.md

```diff
@@ -181,7 +179,7 @@
 - PR #3386 Removing external includes from `column_view.hpp`
 - PR #3369 Add write_partition to dask_cudf to fix to_parquet bug
 - PR #3388 Support getitem with bools when DataFrame has a MultiIndex
-- PR #3408 Fix String and Column (De-)Serialization
```

Comment:
Why did you delete #3408?

Reply:
sorry, was using a new mergetool when i was offline and clearly messed up the CHANGELOG.md, will fix